static/js/apps/nl_interface/app.tsx

```diff
@@ -127,33 +155,55 @@ export function App(): JSX.Element {
             </Container>
           </div>
         </Row>
-        {debugInfo && (
+        {showDebugInfo && (
           <>
             <Row>
-              <b>DEBUGGING INFO: </b>
+              <b>DEBUGGING OPTIONS/INFO: </b>
               <br></br>
             </Row>
             <Row>
-              <b>Execution Status: </b> {debugInfo.status}
-            </Row>
-            <Row>
-              <b>Original Query: </b> {debugInfo.originalQuery}
-            </Row>
-            <Row>
-              <b>Places Detected: </b> {debugInfo.placesDetected.join(", ")}
-            </Row>
-            <Row>
-              <b>Main Place DCID Inferred: </b>
-              {debugInfo.placeDCID}
-            </Row>
-            <Row>
-              <b>Query used for SV detection: </b>
-              {debugInfo.queryWithoutPlaces}
-            </Row>
-            <Row>
-              <b>SVs Matched (with scores):</b>
-              {displaySVMatchScores(debugInfo.svScores)}
+              <label>Embeddings build:</label>
             </Row>
+            <div id="embeddings-build-options">
+              <select
+                value={selectedBuild}
+                onChange={handleEmbeddingsBuildChange}
+              >
+                {buildOptions.map((option) => (
+                  <option key={option.value} value={option.value}>
+                    {option.text}
+                  </option>
+                ))}
+              </select>
+            </div>
+            {debugInfo && (
```

Comment:
This needs to be done in a more React friendly way but that will come in a later update. For now, nothing here has changed except the code has moved location.